frontend/src/components/common/header/header.tsx

```diff
@@ -40,6 +50,16 @@ const Header: React.FC = () => {
                   alt="avatar"
                   loading="lazy"
                 />
+                {isVisible && (
+                  <div className={styles.dropDown}>
```

Comment:
Does this work with the animation (in-out)?

Reply:
I found how to make animations, but its needs a _'react-transition-group'_ lib